app/views/record.html

```diff
@@ -23,7 +35,12 @@
     {% if (record.status | getCanDefer) or (record.status | getCanReinstate) %}
       <div class="record-actions">
         {% if record.status | getCanRecommendForQts %}
-          {% if record | hasOutstandingActions %}
+          {% if not record.courseDetails.startDate | isInPast() %}
```

Comment:
```suggestion
          {% if not record.courseDetails.startDate | isInPast %}
```

Wonder if `isInFuture` would be a helpful filter here.